### PR TITLE
flutter: remove useless examples and add release APK

### PR DESCRIPTION
**Pull request by TheLastGimbus**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

`flutter` page contained some examples that were pretty much useless and obvious, so I removed them, and added two other useful examples

### CODE REVIEW

File: pages/common/flutter.md [hunk 1]
```diff
@@ -31,6 +19,14 @@
 
 `flutter run -d all`
 
+- Download all packages specified in `pubspec.yaml`:
```
Comment (navarroaxel) [inline pages/common/flutter.md:22]:
```suggestion
- Download all the packages listed by the `pubspec.yaml` file in the local path:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Just 1 comment

File: pages/common/flutter.md (hunk 1 above)
Comment (TheLastGimbus) [reply pages/common/flutter.md:22]: They're not "listed by it" - `pubspec` is your configuration file, that *you* specify. This example should be left unchanged :confused:

Comment (sbrl) [reply pages/common/flutter.md:22]: I agree. Let's leave this example as-is.

**Review by sbrl — APPROVED** [review]
Thanks, @TheLastGimbus!

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**